src/Events/Custom_Tables/V1/WP_Query/Custom_Tables_Query.php

```diff
@@ -299,7 +299,7 @@ public function redirect_posts_fields( $request_fields, $query = null ) {
 		 *
 		 * @since 6.0.0
 		 *
-		 * @param string $request_fields The Query fields request, e.g. `ids`.
+		 * @param string   $request_fields The Query fields request, e.g. `ids`.
```

Comment:
```suggestion
		 * @param string $request_fields The Query fields request, e.g. `ids`.
```
Only one space unless we're lining them up.

Reply:
@stratease If you need me to re-approve, LMK